src/components/app/components/Navbar.js

```diff
@@ -40,9 +36,33 @@ const items = [
 
 /*
  * this is the the navigation bar at the top of the home page
- */
-export default class Navbar extends React.Component {
-    render() {
+*/
+class Navbar extends React.Component {
+    logout = ({ key }) => {
+        const { isLoggedIn } = this.props;
+        const { removeCookie } = this.props.actions;
+
+        if (key === SIGN_OUT) {
```

Comment:
this should just be:
```
if (key === SIGN_OUT {
   removeCookie();
}
```
i want you to use the `componentDidUpdate` life cycle to handle the messages.